components/coding/Expandable.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+
+import { PlusCircleIcon } from "@heroicons/react/solid";
+import { MinusCircleIcon } from "@heroicons/react/solid";
+
+interface ExpandableProps {
+  open?: boolean;
+  title: string;
+}
+
+const ExpandableContainer: React.FC<ExpandableProps> = ({
+  open,
+  children,
+  title,
+}) => {
+  const [isOpen, setIsOpen] = useState(open);
```

Comment:
The open prop is currently optional. What would happen if `open` is not supplied and we try to set it as the initial value for isOpen? Typescript isn't complaining so this might be ok, but I think it's worth testing that out to make sure.

Reply:
When open is not supplied it actually defaults to open -- so I think that's what we want it to do more or less.

Reply:
In that case - can we make supplying the open prop mandatory instead of optional? 

If someone was using your component and didn't supply the open prop, it wouldn't have anything to default to. You could add a check to set the initial value to false if it wasn't supplied but that seems more convoluted. 

From a developer standpoint - I think it makes sense to always supply that prop when calling that component

Reply:
that's a good idea.  Just made it mandatory